VERSIONS.md

```diff
@@ -1,6 +1,10 @@
 ## DASTARD Versions
 
-**0.2.7** in progress in April 2020
+**0.2.8** Started September 2020
+* Remove idea of rows/columns from Dastard (use only in `LanceroSource`). Use chan groups otherwise (issue 214).
```

Comment:
use should be used
otherwise seems to imply you still use row/columns sometimes.... maybe Replace with chan groups which are more general.

Reply:
Well, I do use it still, and the only reason is that it goes into the headers of LJH and OFF files. I wish it didn't. Is there some way we could omit that or something? Does MASS really really need that?

If we didn't need the file to know the row/col numbers, then we could simplify a lot. But I fear that we really do need them or want them in the file. How we'll do the analogue in LJH/OFF files for Abaco is another thing that I don't know.

Maybe we can generalize that header, so it basically says "this is chan 4, and it's in a channel group with channels from 1-8". But we are treating files as if they know nothing about array size/geometry. Maybe we should do a phone call?

Reply:
Agreed to continue using row/columns...for now.